docs/reference/module-api/module-overview.md

```diff
@@ -838,19 +838,82 @@ module.exports = {
 };
 ```
 
-### `queries(self)`
+### `queries(self, query)`
+
+The `queries` function is part of the Apostrophe API for querying the database. It takes two arguments: the module (`self`) and the query that will be called (`query`). It must return an object. That object can have two properties:
+<!-- TODO: Link to a real guide on using queries when available. This is simply reference. -->
+
+| `queries` properties | Description |
+| ---- | ---- |
+| [`builders`](#builders) | An object of "builders," or chainable methods, that refine the query or otherwise change its behavior prior to execution. |
+| [`methods`](#methods) | An object of method functions that are used to execute some action on the query. |
```

Comment:
```suggestion
| [`methods`](#methods) | An object of method functions that execute some action on the query. |
```
I think we could ditch "that are used". Might be wrong here. I'm tripping over "An object" "that are".